qos/qos_flow_test.go

```diff
@@ -28,9 +29,9 @@ func TestBuildAuthorizedQosFlowDescriptions(t *testing.T) {
 	authorizedQosFlow := qos.BuildAuthorizedQosFlowDescriptions(smPolicyUpdates)
 
 	fmt.Printf("Authorized QosFlow: %v", authorizedQosFlow)
-	expectedBytes := []byte{0x5, 0x20, 0x45, 0x1, 0x1, 0x5, 0x4, 0x3, 0x6, 0x0,
+	expectedBytes := []byte{0x0, 0x20, 0x45, 0x1, 0x1, 0x5, 0x4, 0x3, 0x6, 0x0,
 		0x65, 0x5, 0x3, 0x6, 0x0, 0xc9, 0x2, 0x3, 0x6, 0x0, 0xb, 0x3, 0x3, 0x6,
-		0x0, 0x15, 0x3, 0x20, 0x45, 0x1, 0x1, 0x3, 0x4, 0x3, 0x6, 0x1, 0x2d, 0x5,
+		0x0, 0x15, 0x0, 0x20, 0x45, 0x1, 0x1, 0x3, 0x4, 0x3, 0x6, 0x1, 0x2d, 0x5,
```

Comment:
Can you explain a bit those changes? I know that changing those values make the test pass, but did we confirm if the production code is right, or the test was right?

Reply:
I took some time to look around the code, and found that those 2 values happen because the test values in `makeSampleQosData` are strings that cannot be cast to ints, returning 0.

Do you think it would be better to change the test data? As it returning 2 QoS flows, maybe we can have one be a proper id, and that would exercise both branches in the code parsing the id.